.ci/system-tests/system_test_Driver.py

```diff
@@ -0,0 +1,477 @@
+# System Test Driver to help with debugging.
+
+import binascii
+from ConfigParser import SafeConfigParser
+import datetime
+import json
+import os
+import shlex
+import subprocess
+import unittest
+import re
+import requests
+import logging
+import pprint
+import hashlib
+import random
+import uuid
+import time
+import sys
+import argparse
+
+from system_test_core import HIRSPortal, AttestationCAPortal, collectors, \
+    send_command, send_command_sha1sum, run_hirs_report, \
+    run_hirs_provisioner_tpm2, parse_xml_with_stripped_namespaces, get_current_timestamp, \
+    get_all_nodes_recursively, touch_random_file_and_remove, get_random_pcr_hex_value, \
+    is_ubuntu_client, is_tpm2, \
+    DEFAULT_IMA_POLICY, DEFAULT_TPM_POLICY
+
+NUMBER_OF_PCRS = 24
+
+suffix = os.environ.get('RANDOM_SYS_TEST_ID')
+if suffix != None:
+    print "Configuring with suffix " + suffix
+    suffix = "-" + suffix
+else:
+    suffix = ""
+
+# Change to point to your HIRS directory
+HOME_DIR = "/HIRS/"
+
+# Change accordingly
+#COLLECTOR_LIST = None
+#COLLECTOR_LIST = ["IMA"]
+#COLLECTOR_LIST = ["TPM"]
+#COLLECTOR_LIST = ["IMA", "TPM"]
+#COLLECTOR_LIST = ["BASE_DELTA_GOOD"]
+COLLECTOR_LIST = ["BASE_DELTA_BAD"]
+
+FORMAT = "%(asctime)-15s %(message)s"
+provisioner_out = None
+
+HIRS_ACA_PROVISIONER_IP="172.19.0.3"
+HIRS_ACA_PROVISIONER_TPM2_IP="172.19.0.4"
+TPM_ENABLED=True
+IMA_ENABLED=False
+
+# Change accordingly
+#HIRS_ACA_PORTAL_IP="172.17.0.2"
+HIRS_ACA_PORTAL_IP="rd8ul-31673ab.dod.mil"
```

Comment:
I thought we weren't going to load system specific information.

Reply:
Your right. This is just a driver I use to make python debugging easier. We can just get rid of the file itself.